subsys/bluetooth/controller/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ zephyr_library_sources(
   )
 
 zephyr_library_sources_ifdef(
-  CONFIG_BT_CTLR_HCI
+  CONFIG_BT_LL_SW_SPLIT_HCI
```

Comment:
> Suggest to retain `BT_CTLR_HCI` and move the block to Controller specific cmake list file.

What file are you referring to? E.g. the `hci` or `ll_sw` subdirectories don't have their own `CMakeLists.txt` files.

What should the Kconfig dependencies of the option be? Right now it's LL_SW, which is wrong based on your description of what this option should be.

Reply:
> What file are you referring to? E.g. the hci or ll_sw subdirectories don't have their own CMakeLists.txt files.

Since there is only one Controller implementation currently in the `subsys/bluetooth/controller` this is in this CMakelists.txt. I do not see these files ever reused by other Controllers (say, blobs with functional Link Layer interface). But a new CMakelists.txt file can be added inside `hci` folder have:

```
add_subdirectory_ifdef(
  CONFIG_BT_CTLR_HCI
  hci
)
```

Vendor specific HCI implementations under `ll_sw/openisa/hci`  etc.. should also be moved from this parent CMakelists.txt into `hci` folder CMakelists.txt.

>What should the Kconfig dependencies of the option be? 

```
    depends on BT_CTLR
```

Reply:
That said, I did fix the dependency like you suggested.